karma.conf.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
```

Comment:
check copyright notices throughout.

Reply:
Wait I am confused here. I thought I needed to go through and change all these to 2021. I seem to be getting conflicting information here about the copyright dates. Can I get clarification on what is correct here?

Reply:
this line changes 2021 to 2020.  Ideally the copyright notices will either be for solely the current year 2021 or a range ending in the current year, e.g. 2020-2021

Have I or anyone else said something that is inconsistent with this?

Reply:
Well I guess I am confused because this library was created in 2021. So I don't understand when I am supposed to use 2020 or 2021.